WebRoot/js/zimbraMail/mail/view/prefs/ZmAccountsPage.js

```diff
@@ -2986,6 +2986,14 @@ function(){
    this.parent._editDelegateButton();
 }
 
+//overriding no results container getter function for custom behavior
+ZmAccountDelegatesListView.prototype._getNoResultsContainer =
+function() {
+	var div = document.createElement("div");
```

Comment:
We are creating a div here not getting an element. Also this is a method that we are overriding from DwtListView and this code is on similar lines that's in there already.